eagle-core/eagle-policy/eagle-policy-base/src/main/java/org/apache/eagle/policy/dao/AlertDefinitionDAOImpl.java

```diff
@@ -83,4 +85,27 @@ public AlertDefinitionDAOImpl(EagleServiceConnector connector){
 			}
 		return map;
 	}
+
+	@Override
+	public void updatePolicyDetails(AlertDefinitionAPIEntity entity, boolean markdownEnabled, String markdownReason) {
+		IEagleServiceClient client = new EagleServiceClientImpl(connector);
+
+		List<AlertDefinitionAPIEntity> entityList = new ArrayList<>();
+		entity.setMarkdownReason(null != markdownReason ? markdownReason : "");
+		entity.setMarkdownEnabled(markdownEnabled);
+		entityList.add(entity);
+
+		try {
+			client.create(entityList, Constants.ALERT_DEFINITION_SERVICE_ENDPOINT_NAME);
+		} catch (IOException | EagleServiceClientException exception) {
+			LOG.error("Exception in updating markdown for policy in HBase", exception.getMessage());
+		} finally {
+			try {
+				if (null != client)
+					client.close();
+			} catch (IOException exception) {
+				LOG.debug("Unable to close Eagle service client, " + exception.getMessage());
```

Comment:
Should the exception be error always? And please make sure not only the message is logged, but also the stack, you might want to pass the exception object to LOG here.